### PR TITLE
Izumi_gate+vground

### PR DESCRIPTION
**Pull request by Trueffeloot**
## CC0 License Agreement

- [X] I agree to the terms of the CC0 License.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ To the best of my knowledge and belief, my contribution is either originally cre
 I understand that this project and its maintainers are not responsible for enforcing the CC0 license, and I release them from any potential liability related to my contribution.
 -->
 
-- [ ] I agree to the terms of the CC0 License.
+- [X] I agree to the terms of the CC0 License.
```
Comment (TakaRikka) [inline .github/pull_request_template.md:12]: this doesn't need to be changed in the file itself, it's only used within github. you can revert the change here

File: rel/d/a/obj/d_a_obj_vground/d_a_obj_vground.cpp [hunk 2]
```diff
@@ -1,357 +1,163 @@
...
+            return 0;
+        } else {
+            J3DAnmTevRegKey* mpTevReqKey = (J3DAnmTevRegKey*)dComIfG_getObjectRes(l_arcName, 8);
+            mpBrk = new mDoExt_brkAnm();
+            if ((!mpBrk) || (!mpBrk->init(mpObjectRes, mpTevReqKey, 1, NULL, 1.0f, NULL, -1))) {
+                return 0;
+            }
+            return 1;
+        }
+    }
```
Comment (TakaRikka) [inline rel/d/a/obj/d_a_obj_vground/d_a_obj_vground.cpp:61]: readability can be improved by removing unnecessary nesting. for example, an else statement if not necessary after the returns. you can just do it like

```c++
if (!mpModel) {
    return 0;
}

J3DAnmTextureSRTKey* mpTextureSRTKey = (J3DAnmTextureSRTKey*)dComIfG_getObjectRes(l_arcName, 0xb);
```
and so on so forth for the others.

**Comment by Trueffeloot** [comment]
Function should look better now 👍 I hope it's good like this

**Comment by Trueffeloot** [comment]
Oh yeah and I reverted the .md file

**Review by TakaRikka — APPROVED** [review]
